specs/http-client2/index.html

```diff
@@ -255,14 +285,23 @@ <h3>Request Options</h3>
         </p>
 
         <dl>
-          <dt><code>status-only</code> (<code>xs:boolean</code>)</dt>
-          <dd>Skip body in response. Default: false.</dd>
+          <dt><code>http-version</code> (<code>xs:string</code>)</dt>
+          <dd>The HTTP Version to use for the request. Default: 1.1. Supported versions are <code>1.0</code> and <code>1.1</code>. An unsupported version raises the error: TODO-ERROR-CODE.</dd>
+
+          <dt><code>headers</code> (<code>map(xs:string, xs:string)</code>)</dt>
+          <dd>HTTP header fields.</dd>
+
+          <dt><code>content-encoding</code> (<code>xs:string</code>)</dt>
+          <dd>Uses a specific content encoding for the request body. Supported encodings are <code>gzip</code> and <code>deflate</code>. An usupported encoding raises the error: TODO-ERROR-CODE.</dd>
+
+          <dt><code>chunked</code> (<code>xs:boolean</code>)</dt>
+          <dt>Use chunked Transfer Encoding. Default: true for HTTP 1.1, false for HTTP 1.0. Sets the HTTP Header <code>Transfer-Encoding: chunked</code>, overrides any existing header in <code>$options</code>.</dt>
 
           <dt><code>follow-redirect</code> (<code>xs:boolean</code>)</dt>
           <dd>Follow HTTP redirects. Default: true.</dd>
 
           <dt><code>timeout</code> (<code>xs:integer</code>)</dt>
-          <dd>Maximum number of seconds to wait for a response.</dd>
```

Comment:
I would prefer to stick with `seconds` (to stay compliant with version 1.0; http://expath.org/spec/http-client#d2e178).

Reply:
hmmm, in this day and age, I would argue that seconds does not offer enough granularity for fast applications. I can *live* with `seconds`, but I would *prefer* `milliseconds`.

Considering we are breaking backwards compatibility all over the place, e.g. with how options and requests and responses are handled. Does it matter here? Maybe we should just throw backwards compatibility out. Users can always use the 1.0 HTTP Client if they wish instead ;-)

Reply:
I have some concerns if the key is named exactly the same. Maybe we could use `xs:decimal` as data type? This would allow people to upgrade their code more easily.

Reply:
An interesting option :-)